de.desy.constellation.yml

```diff
@@ -0,0 +1,41 @@
+id: de.desy.constellation
+runtime: org.kde.Platform
+runtime-version: "6.8"
+sdk: org.kde.Sdk
+command: Satellite
+finish-args:
+  - --share=network
+  - --filesystem=host
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /share/pkgconfig
+modules:
+  - name: Constellation
+    buildsystem: meson
+    config-opts:
+      - -Dbuildtype=release
+      - -Dimpl_py=disabled
+      - -Dcxx_tools=false
+      - -Dcxx_tests=disabled
+      - -Dcontroller_missioncontrol=true
+      - -Dlistener_observatory=true
+      - -Dsatellite_dev_null_receiver=true
+      - -Dsatellite_eudaq_native_writer=true
+      - -Dsatellite_random_transmitter=true
+      - -Dsatellite_sputnik=true
+    sources:
+      - type: archive
+        url: https://gitlab.desy.de/constellation/constellation/-/releases/v0.3/downloads/Constellation-v0.3.tar.xz
+        sha256: 8d30f5c7a1203ded7768a00e58281fffc907abfd1257a50de0b533f99a56ddad
+  - name: Constellation-metainfo
```

Comment:
no need for a separate module. this should be merged with the "main" one.

Reply:
I.e. merged upstream? We can do that, but not retroactively for this release - can we fix this in the next release?

Reply:
yes having the appstream file upstream for a future release would be great but that's not what I mean here. There is no reason to have s separate module to install the appstream file.
(the module list start line 17)

Reply:
Right, but how do I then install the metainfo file? Can I add build commands to meson?

Reply:
and metainfo should be installed by meson, not manually